egs/iwslt18st/st1/path.sh

```diff
@@ -3,11 +3,12 @@ KALDI_ROOT=$MAIN_ROOT/tools/kaldi
 
 [ -f $KALDI_ROOT/tools/env.sh ] && . $KALDI_ROOT/tools/env.sh
```

Comment:
can you check `path.sh` in the other recipe? We should follow the style similar to others as much as possible.

Reply:
@sw005320 I'm sorry. I forgot to follow this direction. I just fixed it.